situp-plugins/common/src/main/java/com/amazon/situp/plugins/processor/StringProcessor.java

```diff
@@ -13,9 +13,11 @@
  * An simple String implementation of {@link Processor} which generates new Records with upper case content. The current
  * simpler implementation does not handle errors (if any).
```

Comment:
Docs are no longer in sync - can now lowercase strings depending on configuration.

Reply:
Good catch!